app/views/suggestion_mailer/person_email.html.haml

```diff
@@ -0,0 +1,36 @@
+%table
+  %tbody
+    %tr
+      %td{width: "20"}
+      %td{width: "560"}
+        %table
+          %tbody
+            %tr
+              %td{ style: "padding-bottom: 20px;", width: "560"}
+                Hello,
+            %tr
+              %td{ style: "line-height: 30px; padding-bottom: 20px;", width: "560"}
+                %p
+                  = @suggester.name
+                  has suggested some improvements to your
+                  PeopleFinder profile.
+                - if @suggestion.missing_fields_info.present?
+                  %p
+                    They provided the following information about missing fields:
+                  %p
+                    = @suggestion.missing_fields_info
+                - if @suggestion.incorrect_fields_info.any?
+                  %p
+                    They suggested that the following fields were incorrect:
+                  %ol
+                    - @suggestion.incorrect_fields_info.each do |field|
+                      %li= field
+            %tr
+              %td{ :style => "padding-bottom: 40px;", width: "560"}
```

Comment:
Very minor thing, but the HAML attribute format is inconsistent. This line and line 34 below use `:attribute => "value"` (rocket syntax) whereas above inline style declarations use `attribute: "value"`. If there's a reason for that, then cool, but otherwise consistency would be good for maintainability. No biggie though.